sympy/ntheory/primetest.py

```diff
@@ -110,17 +110,22 @@ def mr(n, bases):
     return True
 
 
-def isprimeE(n,b):
-	"""isprimeE(n) - Test whether n is prime or an Euler pseudoprime to base b."""
-	if (not mr(n,b)): return False
+def isprimeE(n, b):
+    """isprimeE(n ,b) - Test whether n is prime or an Euler pseudoprime to base b."""
```

Comment:
@normalhuman gave you feedback that is consistent with our desire for code in the SymPy codebase. Please see [theis PEP](https://www.python.org/dev/peps/pep-0257/) if you have questions about docstrings.  While the code is what "get's it done" the docstring is the welcome mat for the public. Your docstring is fine past the first space after the dash.

Reply:
done.
thanks for review..